test/Constraints/variadic_generic_types.swift

```diff
@@ -29,3 +29,18 @@ struct MissingMemberError<each T> {
     // expected-error@-1 {{value of type 'MissingMemberError<repeat each T>' has no member 'doesNotExist'}}
   }
 }
+
+// https://github.com/apple/swift/issues/66095
+do {
+  struct Test<each S> {
+    init(_ s: repeat each S) {}
+  }
+
+  func test1<each T>(_ v: repeat each T) -> Test<repeat each T> {
+    return Test(repeat each v) // Ok
+  }
+
+  func test2<each T>(_ v: repeat each T) -> Test<repeat each T> {
```

Comment:
Could you please also add a few tests that append concrete pack elements onto either side of the forwarded pack, e.g. `Test<repeat each T, Int, String>` and `Test<Int, String, repeat each T>`?

Reply:
Sure!

Reply:
✅